### ISSUE TITLE
Taking over flower

### ISSUE THREAD

**Issue by GitRon** [jazzband/help#293]
Hi there!

The celery monitoring tool [flower](https://pypi.org/project/flower/) is currently abandoned and in a state where the latest pypi verison is not working anymore. The maintainer fixed it but is not pushing the changes. We are currently discussing how to save the project [in this issue](https://github.com/mher/flower/issues/1211#issuecomment-1149773129).

If we'd mange to get access to the project, would it be suitable to be maintained by Jazzband?

Thx!  
Ronny

**Comment by jezdez** [comment]
Yeah, this seems like a reasonable project, but please make sure to read the docs about [releases](https://jazzband.co/about/releases), [guidelines](https://jazzband.co/about/guidelines), [security](https://jazzband.co/about/security) and [FAQ](https://jazzband.co/about/faq). And a gentle reminder that you'd agree to follow the Jazzband [Code of Conduct](https://jazzband.co/about/conduct).

And just to clarify: @mher would need to confirm here that that's what they want.

Afterwards feel free to transfer the project using the regular repository transfer feature (join Jazzband before on https://jazzband.co/) to the Jazzband org. A TODO issue is automatically created upon transfer to implement the Jazzband guidelines.

**Comment by GitRon** [comment]
Thanks for the update! Maybe @mher could say if this would be ok for him?
